File: Java/SETPractice/src/main/java/JavaPractice/ExtentionMethods/ExtentionMethodTest.java
package JavaPractice.ExtentionMethods;

public class ExtentionMethodTest {
    public static void main(String[] args) {
        Logging log = new Logging();
        InfoLogging.InfoLog(log)
                .PrintLog("This is my info log");

        InfoLogging.InfoLog(log)
                .PrintLog("This is my another info log");

        ErrorLogging.ErrorLog(log)
                .PrintLog("This is error log");

    }
}

File: DotNet/SETPractice/SETPractice/CSharpPracticeTests/ExtentionMethodTest.cs
using CSharpPractice.ExtentionMethods;
using NUnit.Framework;

namespace SETPractice.CSharpPracticeTests
{
    [TestFixture]
    public class ExtentionMethodTest
    {
        [Test]
        public void ExtentionMtdTest()
        {
            Logging log = new Logging();
            log.InfoLog();
            log.PrintLog("This is info log");
            log.InfoLog();
            log.PrintLog("This is another info log");

            InfoLogging.InfoLog(log);
            log.PrintLog("");
        }
    }
}
